src/card/card.jsx

```diff
@@ -53,21 +61,32 @@ const Card = React.createClass({
   },
 
   getInitialState() {
+    const uncontrolledExpanded = this.props.initiallyExpanded ? true : false;
```

Comment:
That seems useless (http://eslint.org/docs/rules/no-unneeded-ternary) `initiallyExpanded` is already a bool.